src/SourceBuild/content/eng/pipelines/vmr-license-scan.yml

```diff
@@ -111,23 +115,14 @@ extends:
             artifactName: $(Agent.JobName)_BuildLogs_Attempt$(System.JobAttempt)
         steps:
 
-        - script: |
-            source ./eng/common/tools.sh
-            InitializeDotNetCli true
-          displayName: Install .NET SDK
-          workingDirectory: $(Build.SourcesDirectory)
-
         - task: PipAuthenticate@1
           displayName: 'Pip Authenticate'
           inputs:
             artifactFeeds: public/dotnet-public-pypi
             onlyAddExtraIndex: false
```

Comment:
This isn't needed anymore.

Reply:
Got it. Update: https://github.com/dotnet/dotnet/pull/253